myproject/blog/models.py

```diff
@@ -1,44 +1,41 @@
 from django.db import models
-from django.utils import timezone
 from django.contrib.auth.models import User
 
 class BaseModel(models.Model):
-    created_at = models.DateTimeField(default=timezone.now, editable=False)
+    created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
 
     class Meta:
         abstract = True
-        db_table = "base_model"
 
-class Author(BaseModel):
+class Authors(BaseModel):
```

Comment:
This need to be singular i.e. `Author`, only table name will be in plural which is `db_table` (Authors).